sdk/keyvault/keyvault-keys/tests.yml

```diff
@@ -10,13 +10,26 @@ stages:
       # against multiple platforms in parallel (we're limited to a single
```

Comment:
I think this comment needs to be moved down to where AdditionalMatrixConfigs is defined

Reply:
Ok, I have moved this comment to where AdditionalMatrixConfigs is defined, please have a look.